schema/default_meta.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://burrito.bible/schema/default_meta.schema.json",
+    "title": "Scripture Burrito Meta (Default)",
+    "type": "object",
+    "description": "Information about the Scripture Burrito metadata file (default variant).",
+    "properties": {
+        "variant": {
+            "type": "string",
+            "pattern": "^default$",
```

Comment:
Let's make this `"const": "default"` instead.  I know you're a big fan of regexes, but this one doesn't need it. ;-)

Reply:
I was allowing for future expansion in case we find other spellings like dephault.